index.html

```diff
@@ -896,7 +898,14 @@
           [SameObject] readonly attribute PushSubscriptionOptions options;
           ArrayBuffer? getKey(PushEncryptionKeyName name);
           Promise&lt;boolean&gt; unsubscribe();
-          serializer;
+
+          PushSubscriptionJSON toJSON();
+        };
+
+        dictionary PushSubscriptionJSON {
+          USVString endpoint;
+          DOMTimeStamp? expirationTime;
+          record keys;
```

Comment:
Ah yeah, I forgot to follow up on that - ReSpec seems to choke on `record<K, V>` syntax for some reason. The key should be a `DOMString` since it's one of the `PushEncryptionKeyName` values, and enum values are defined to be `DOMString`s.